src/els_parser.erl

```diff
@@ -181,22 +181,29 @@ do_points_of_interest(Tree, EndLocation) ->
 application(Tree) ->
   case application_mfa(Tree) of
     undefined -> [];
-    MFA -> [poi(erl_syntax:get_pos(Tree), application, MFA)]
+    {F, A} ->
+      Pos = erl_syntax:get_pos(Tree),
+      case is_erlang_bif(F, A) of
+        %% Call to a function from the `erlang` module
+        true -> [poi(Pos, application, {erlang, F, A}, #{imported => true})];
+        %% Local call
+        false -> [poi(Pos, application, {F, A})]
+      end;
+    MFA ->
+      [poi(erl_syntax:get_pos(Tree), application, MFA)]
   end.
 
+-spec is_erlang_bif(atom(), arity()) -> boolean().
+is_erlang_bif(F, A) ->
+  lists:member({F, A}, erlang:module_info(exports)).
```

Comment:
This is not 100% accurate. Not all functions in the `erlang` module are automatically imported. We probably should use [`erl_internal:bif/2`](https://erlang.org/doc/man/erl_internal.html#bif-2) here

Reply:
Created #633 as a follow-up.